integration/nwo/components.go

```diff
@@ -32,7 +34,7 @@ func (c *Components) Discover() string {
 }
 
 func (c *Components) Idemixgen() string {
-	return c.Build("github.com/hyperledger/fabric/cmd/idemixgen")
```

Comment:
why don't we build it anymore though? can't we do `go install` or something instead?

Reply:
you mean we could go get the dependency in the makefile and then build it here like we do the others? Yeah, that actually isn't a bad idea.. I'll do it - thx

Reply:
Yeah something like that... but I tried with `go install` and it didn't like it

Reply:
Fixed now - wdyt?